instrumentation-security/cassandra-datastax-3/src/main/java/com/datastax/driver/core/SessionManager_Instrumentation.java

```diff
@@ -0,0 +1,38 @@
+package com.datastax.driver.core;
+
+import com.datastax.driver.core.querybuilder.CassandraUtils;
+import com.newrelic.api.agent.security.NewRelicSecurity;
+import com.newrelic.api.agent.security.schema.AbstractOperation;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+
+@Weave(type = MatchType.ExactClass, originalName = "com.datastax.driver.core.SessionManager")
+abstract class SessionManager_Instrumentation {
+    final Cluster cluster = Weaver.callOriginal();
```

Comment:
Please use getter instead of class member

Reply:
Fixed in [commit](https://github.com/newrelic/csec-java-agent/pull/70/commits/83d2181cbef1e2ea56fc5d4dd1aa429d0afd51b1)